modules/nf-core/dshbio/filterbed/tests/main.nf.test

```diff
@@ -18,7 +18,7 @@ nextflow_process {
                 """
                 input[0] = [
                     [ id:'test' ], // meta map
-				    [ file(params.modules_testdata_base_path + 'genomics/sarscov2/genome/bed/test.bed', checkIfExists: true) ]
+				    [ file(params.modules_testdata_base_path + 'genomics/homo_sapiens/genome/chr21/sequence/multi_intervals.bed', checkIfExists: true) ]
```

Comment:
what was the problem with the old data set?

Reply:
I don't know whether to laugh or to cry :wink:

In version 2.4 &rarr; 3.0 BED parsing was updated to strictly follow the BED version 1 specification
https://github.com/samtools/hts-specs/blob/master/BEDv1.pdf

and improve standing per this paper

Assessing and assuring interoperability of a genomics file format
https://www.biorxiv.org/content/10.1101/2022.01.07.475366v1

but then several of the BED files included in our test data are actually invalid, per the specification.